index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   excluded: [
     'node_modules',
+    'ios/Pods',
```

Comment:
```suggestion
    'Pods',
```

Does it work just with `Pods`? This would be more portable (for non-Capacitor projects).

Reply:
no, it doesn't

Reply:
😢